### PR TITLE
Merge changes from Blivet and Anaconda

### PR DESCRIPTION
**Pull request by vojtechtrefny**
We accumulated some changes in Anaconda and Blivet that should be merged here. @jkonecny12 @VladimirSlavik we should stop doing "downstream" changes to translation canary and make sure to merge everything here first.

### CODE REVIEW

File: tests/project-tests/test_projects.sh [hunk 1]
```diff
@@ -30,15 +30,8 @@ while read project_name branch ; do
...
-    zanata_output="$(zanata pull --project-type gettext --project-id "$project_name" --project-version "$branch" --transdir "$podir" --url https://fedora.zanata.org/ 2>&1)"
-    if [ $? -ne 0 ]; then
-        echo "Zanata pull failed for $project_name:$branch"
-        echo "$zanata_output"
-        status=1
-        rm -rf "$podir"
-        continue
-    fi
+    # Download translations
+    git clone --depth 1 -b $branch -- "https://github.com/rhinstaller/anaconda-l10n.git" $podir
```
Comment (jkonecny12) [inline tests/project-tests/test_projects.sh]: This should not be hard-coded to Anaconda git. Let's use ``$project_name`` or change this other way around.

**Review by jkonecny12 — CHANGES_REQUESTED** [review]
Looks good to me otherwise.

**Comment by jkonecny12** [comment]
And thanks a lot for doing this. It would be great to have these on one place!

File: tests/project-tests/test_projects.sh (hunk 1 above)
Comment (vojtechtrefny) [reply tests/project-tests/test_projects.sh]: Ok, I've changed this to run tests on the `rhinstaller/*-l10n` repos. I'm not really sure what was the intention behind this test/make target, it' not really a test case for translation canary, it looks more like a helper target to run translation canary against all rhinstaller projects. The test itself is now failing, but the errors look like legit errors in the PO files for anaconda and python-meh so I assume it works as expected.

**Comment by VladimirSlavik** [comment]
What is the intended relationship between this project and the translation-canary dir in somewhere? It would be nice to use this, instead of manually copying changes...

**Comment by vojtechtrefny** [comment]
> What is the intended relationship between this project and the translation-canary dir in somewhere? It would be nice to use this, instead of manually copying changes...

This project should be used as a git subtree in our projects, but we didn't do this correctly in past few years and instead of pushing changes here and merging them to anaconda/blivet/etc. we've added changes locally... This is an attempt to merge all these changes here and after that we should all "force rebase" our local subtrees to the "upstream" version.

(Using submodule would make this easier IMHO but the original author [didn't think so](https://github.com/rhinstaller/anaconda/pull/463#issuecomment-160985911). Maybe I'm just a fan of submodules :-))

**Comment by VladimirSlavik** [comment]
I also thought along these lines...

**Comment by jkonecny12** [comment]
> > What is the intended relationship between this project and the translation-canary dir in somewhere? It would be nice to use this, instead of manually copying changes...
> 
> This project should be used as a git subtree in our projects, but we didn't do this correctly in past few years and instead of pushing changes here and merging them to anaconda/blivet/etc. we've added changes locally... This is an attempt to merge all these changes here and after that we should all "force rebase" our local subtrees to the "upstream" version.
> 
> (Using submodule would make this easier IMHO but the original author [didn't think so](https://github.com/rhinstaller/anaconda/pull/463#issuecomment-160985911). Maybe I'm just a fan of submodules :-))

I'm fan of submodules too but seems like the only one on our team :D

**Review by jkonecny12 — APPROVED** [review]
Looks great to me now. Thanks!

**Comment by jkonecny12** [comment]
Also I fixed errors in Anaconda translations thanks to this.

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

**Pull request merged**